<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Моделирование заправки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="teLog">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="teCars">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblCarsNum">
         <property name="text">
          <string>Число машин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leCarsNum">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblGenExpVal">
         <property name="text">
          <string>Матожидание генератора</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leGenExpVal">
         <property name="text">
          <string>30</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblProcExpVal">
         <property name="text">
          <string>Матожидание заправки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leProcExpVal">
         <property name="text">
          <string>30</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblProcDiap">
         <property name="text">
          <string>Полуинтервал заправки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leProcDiap">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRowsNum">
         <property name="text">
          <string>Число рядов колонок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leRowsNum">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblStationsInRowNum">
         <property name="text">
          <string>Число колонок в ряду</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leStationsInRowNum">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbEvent">
         <property name="text">
          <string>Событийная протяжка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAvgTimeInQueue">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblAvgTimeToLeave">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFreedTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblWastedTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblModelingTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
